Mathlib/FieldTheory/CardinalEmb.lean

```diff
@@ -122,6 +122,7 @@ def leastExt : ι → ι :=
       rw [adjoin_basis_eq_top, ← eq_top_iff] at this
       apply_fun Module.rank F at this
       refine ne_of_lt ?_ this
+      let _ : AddCommMonoid (⊤ : IntermediateField F E) := inferInstance
```

Comment:
I fear that if you don't add a comment explaining why this line is there, it will be deleted again in the near future.

Reply:
Whoever removes it should feel a big slowdown (likely a timeout). If not, why should we block the removal?
If there's a tracking issue for such slow instances on CoeSort of subobjects, it would be appropriate to refer to it, so as to remind us to remove this `let _` when the issue is fixed.